src/main/java/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -91,6 +104,20 @@ public final Connection makeConnection() throws SQLException {
         }
     }
 
+    private HikariDataSource hikariDataSource;
+
+    public void createDataSource() {
+        HikariConfig config = new HikariConfig();
+        config.setJdbcUrl(workConf.getUrl());
+        if (!StringUtils.isEmpty(workConf.getUsername())) {
+            config.setUsername(workConf.getUsername());
+            config.setPassword(workConf.getPassword());
+            config.setMaximumPoolSize(workConf.getTerminals());
```

Comment:
There are some extra properties in the tpcc code. Can they be set using the jdbc url?

Reply:
TPCC does not take jdbc url as input in the workload config.
I have kept it same is of the default benchbase, rest of the properties can be set through jdbc url.